website/.env

```
@@ -5,6 +5,7 @@ PUBLIC_GITHUB_SPONSORS_URL="https://github.com/sponsors/fabian-hiller"
 
 PUBLIC_STACKBLITZ_ANGULAR_URL="https://stackblitz.com/edit/formisch-playground-angular"
 PUBLIC_STACKBLITZ_REACT_URL="https://stackblitz.com/edit/formisch-playground-react"
+PUBLIC_STACKBLITZ_REACT_NATIVE_URL="https://stackblitz.com/edit/formisch-playground-react-native"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)